pages/cultural-dimensions.py

```diff
@@ -5,12 +5,27 @@
 
 dash.register_page(
     __name__,
-    description="This app uses tabs to highlight cultural dimensions on a bar chart.",
+    description="This app uses tabs to highlight cultural dimensions on a bar chart. It has a top-bottom layout and a regular-callback.",
```

Comment:
This is actually the only app I think that has a side-by-side layout. Can you please update the second sentence of the description to be :
It has a side-by-side layout and a regular-callback.